### DISCUSSION TITLE
Adding beta releases to F-Droid

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [florisboard/florisboard#1314] [Q&A]
What do you think about adding beta releases to F-Droid?

I tried grabbing a beta from there after reviewing the source code, and was surprised it wasn't available.

**Comment by patrickgold** [comment]
You can grab a beta build from F-Droid via the IzzyOnDroid F-Droid repository. "Stable" builds are published on the main F-Droid repo. Hope that helps!

**Reply by Gitoffthelawn** [reply]
Thank you.  That does help.

Just FYI, for some projects, including the F-Droid client itself, beta versions (and even alpha versions) are available via the F-Droid client alongside "stable" builds.  I'm pretty sure you get to tag which version gets recommended/pushed to users.
